packages/paste-website/src/pages/components/popover/index.mdx

```diff
@@ -75,6 +78,19 @@ export const pageQuery = graphql`
 The Popover component is a non-modal dialog that is commonly used for displaying additional rich content
 above your UI.
 
+### Popover compared to Tooltip
+
+<div style="position: relative; padding-bottom: 56.25%; height: 0;">
```

Comment:
Are you trying to maintain aspect ratio but not using your aspect ratio component?

Reply:
I guess it is, this was just a copy/pasted embed code from Loom.

Reply:
It'd be worth seeing if Aspect-Ratio works here. I don't think we've used it too often in the wild.